src/bootstrap/Cargo.lock

```diff
@@ -0,0 +1,459 @@
+# This file is automatically @generated by Cargo.
```

Comment:
Please add `src/bootstrap/Cargo.lock` to the list of `Cargo.lock` files tidy looks at.